examples/make-db-migration.rs

```diff
@@ -563,6 +624,118 @@ pub fn wasm_migration(updates: &mut Vec<migrations::DbUpdateType>) {
     });
 }
 
+/// Demonstrate how to set the shielded reward parameters of IBC tokens using a
+/// migration
+pub fn shielded_reward_parameters_migration(
+    updates: &mut Vec<migrations::DbUpdateType>,
+) {
+    const TOKENS: [(
+        Denomination,
+        TokenAddress,
+        MaxRewardRate,
+        TargetLockedAmount,
+        KpGain,
+        KdGain,
+    ); 6] = [
+        (
+            Denomination(0),
```

Comment:
Same here, to my understanding the denomination of ibc assets should always be 0